xscen/io.py

```diff
@@ -358,6 +359,7 @@ def save_to_zarr(
     encoding: dict = None,
     mode: str = "f",
     itervar: bool = False,
+    timeout_cleanup: bool = False,
```

Comment:
```suggestion
    timeout_cleanup: bool = True,
```

Are there cases where we don't want that? It seems to be a pretty useful QoL to prevent corrupted variables. Unless there are such situations, I'd say we either put the default as True, or just do it without adding it as a n argument to the function.